app/views/communities/show.html.erb

```diff
@@ -2,54 +2,85 @@
   <%= image_tag(@community.cover_image, height: "200px", width: "1200px") %>
 <% else %>
   <%= link_to "#" do %>
-    <img height= "200px", width= "1200px" src="https://cdn.cbeditz.com/cbeditz/preview/solid-dark-blue-gradient-background-wallpapers-70-116145018144hkixikfuh.jpg">
+    <img height= "170px", width= "100%" src="https://cdn.cbeditz.com/cbeditz/preview/solid-dark-blue-gradient-background-wallpapers-70-116145018144hkixikfuh.jpg">
   <% end %>
 <% end %>
-
-
-<div class="row mb-4">
-  <div class="col-sm-8">
-    <div class = "details">
-    <% if @community.profile_image.attached? %>  
-      <%= image_tag(@community.profile_image, class: "profile-pic") %>
-    <% else %>
-      <%= link_to "#" do %>
-        <img class="profile-pic" src="https://www.iconpacks.net/icons/2/free-reddit-logo-icon-2436-thumb.png">
+<div class="container">
+  <div class="row">
+    <div class="col-sm-1 mr-3">
+      <% if @community.profile_image.attached? %>
+        <%= image_tag(@community.profile_image, class: "profile-pic") %>
+      <% else %>
+        <%= link_to "#" do %>
+          <img class="profile-pic" src="https://www.iconpacks.net/icons/2/free-reddit-logo-icon-2436-thumb.png" class="profile-pic">
```

Comment:
don't use external url for assets, instead download the image and load it locally